src/dali/plugin/input/csv/blockfi.py

```diff
@@ -152,14 +154,14 @@ def load(self) -> List[AbstractTransaction]:
                             asset=line[self.__CURRENCY_INDEX],
                             exchange=self.__BLOCKFI,
                             holder=self.account_holder,
-                            transaction_type="Sell",
+                            transaction_type=Keyword.SELL.value,
                             spot_price=Keyword.UNKNOWN.value,
                             crypto_out_no_fee=str(-RP2Decimal(line[self.__AMOUNT_INDEX])),
                             crypto_fee="0",
                             notes="ACH withdrawal",
                         )
                     )
-                elif transaction_type == _WITHDRAWAL:
+                elif transaction_type in [_WITHDRAWAL, _BIA_WITHDRAWAL]:
```

Comment:
Just curious: what's the difference between BIA withdrawal and vanilla withdrawal?

Reply:
Oh, I just realized: it's withdrawal from their interest account.

Reply:
My understanding is that this type of transaction is just an internal transfer from the interest account to the normal account (at least that's how it shows in my case): if that's correct, this is not a real withdrawal, because the amount never leaves BlockFi (in this case it would be better to just ignore "BIA Withdraw"). Can you verify this?

Reply:
After looking at my CSV report more closely I'm seeing the same thing:

```
USDC,-21.19100000,Withdrawal,2022-06-15 20:29:37
USDC,21.19100000,BIA Withdraw,2022-06-14 00:52:19
USDC,21.19198526,Interest Payment,2022-05-31 23:59:59
```

I think we can safely just ignore and `DEBUG` log the BIA withdrawal since it is just a transfer between internal blockfi accounts, right?

Reply:
I agree: we can ignore it for the purposes of RP2 calculations.